usbtmc/usbtmc.py

```diff
@@ -696,51 +718,77 @@ def clear(self):
             raise UsbtmcException("Clear failed", 'clear')
 
     def remote(self):
-        "Send remote command"
+        """
+        Send the remote command to the instrument.
+        :return:
+        """
         raise NotImplementedError()
 
     def local(self):
-        "Send local command"
+        """
+        Send the local command to the instrument.
+        :return:
+        """
         raise NotImplementedError()
 
     def lock(self):
-        "Send lock command"
-
+        """
+        Advantest/ADCMT ONLY: Send the lock command to the instrument.
```

Comment:
Since these were added as part of a pull request, that would break those folks using it. For this PR, let's leave it, and perhaps put a feeler out to the original requestor about changing this in a subsequent PR.